app/konnect/getting-started/configure-mesh.md

```diff
@@ -33,15 +33,14 @@ You now have a {{site.mesh_product_name}} global control plane. This control pla
 
 After creating the global control plane, you must add a zone to that control plane. Adding a zone allows you to manage services added to that zone and send and receive configuration changes to the zone. 
 
-1. In {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.
-1. Click **Create Zone**. 
-1. Enter "zone-1" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
+1. From the `example-cp` control plane you just created, click **Create Zone**. 
+1. Enter "zone-1" in the **Zone name** field for the new zone, and then click **Create Zone & generate token**.
```

Comment:
@johnharris85 In this step, they can select Kubernetes or Universal. I think they have to select Kubernetes so that it will work with the demo app later, is that correct? If so, I'd like to mention that they need to select Kubernetes in the step.

Reply:
Yes as part of the getting started, it's tailored for Kubernetes. We may eventually provide a universal getting started in a tabbed box or something, but K8s only for now.